src/Orleans/Async/Cancellation/GrainCancellationToken.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Orleans.CodeGeneration;
+using Orleans.Runtime;
+using Orleans.Serialization;
+
+namespace Orleans.Async
+{
+    /// <summary>
+    /// Grain cancellation token that can be passed thought cross-domain boundaries
+    /// </summary>
+    [Serializable]
```

Comment:
It's not about binary formatter and the comment about domain is misleading. It's a regular cross grain calls object. We have an internal convention to mark any object that can be send around as serializable, even if this is redundant. Look at any other internal object that us sent: steam ID, queue ID, ... 
So its just keeping the convention. We should change the comment not to talk about domains.